schedule/sle-micro/dvd.yaml

```diff
@@ -28,6 +28,7 @@ schedule:
   - microos/libzypp_config
   - microos/one_line_checks
   - microos/services_enabled
+  - microos/cockpit_check
```

Comment:
Pretty much every module checks something, so the `_check` suffix is IMO redundant

Reply:
We plan to have a test for cockpit itself, with selenium for UI testing. That will be called `cockpit` :)

Reply:
Then maybe `cockpit_service` or `cockpit_basic`?